app/views/shared/proposals/_tags_form.html.haml

```diff
@@ -1,5 +1,6 @@
+%h3 Review Tags
 %fieldset
-  - if event.review_tags.any?
+  - if event.public_tags?
```

Comment:
I've move this check to wherever this partial is included as everything in here shouldn't show up unless tags are on.